.github/CHANGELOG.md

```diff
@@ -28,6 +31,9 @@
 * Loosen the upper-bound on `thewalrus` and upgrade it.
   [(#454)](https://github.com/XanaduAI/MrMustard/pull/454)
 
+* Update referenced TensorFlow version to `v2.17.0`
+  [(#506)](https://github.com/XanaduAI/MrMustard/pull/506)
+
```

Comment:
We don't need to update TF version in this PR as the fix preserves the support for TF >= 2.15.

```suggestion
```